pyproject.toml

```diff
@@ -30,7 +30,8 @@ classifiers = [
 ]
 dependencies = ["databricks-labs-blueprint>=0.9.1,<0.10",
                 "databricks-sdk~=0.30",
-                "databricks-labs-lsql>=0.5,<0.13"]
+                "databricks-labs-lsql>=0.5,<0.13",
+                "pyspark~=3.5.0"]
```

Comment:
Technically we need it only for Development, not for the runtime. I'm afraid about installing with `pip -U ...` on the cluster